backend/build.gradle

```diff
@@ -1,77 +1,65 @@
-apply plugin: 'kotlin'
+apply plugin: 'kotlin-platform-jvm'
+apply plugin: 'kotlinx-serialization'
+
 apply plugin: 'application'
-apply plugin: "com.github.johnrengelman.shadow"
+apply plugin: 'com.github.johnrengelman.shadow'
 
-mainClassName = "org.jetbrains.kotlinconf.backend.ServerKt"
+mainClassName = 'org.jetbrains.kotlinconf.backend.ServerKt'
 
 sourceCompatibility = 1.8
 
-kotlin {
-    experimental {
-        coroutines "enable"
-    }
-}
+dependencies {
+    expectedBy project(":common")
 
-repositories {
-    mavenCentral()
-    mavenLocal()
-    maven { url 'http://dl.bintray.com/kotlin/kotlin-eap' }
-    maven { url "https://dl.bintray.com/kotlin/kotlinx" }
-    maven { url "https://dl.bintray.com/kotlin/ktor" }
-    maven { url "https://dl.bintray.com/sargunster/maven" }
-    maven { url "https://dl.bintray.com/kotlin/squash" }
-}
+    implementation "io.ktor:ktor-client-apache:$ktor_version"
+    implementation "io.ktor:ktor-client-json-jvm:$ktor_version"
 
-dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
     implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
-    implementation project(":common-jvm")
     implementation "io.ktor:ktor-server-netty:$ktor_version"
     implementation "io.ktor:ktor-gson:$ktor_version"
     implementation "io.ktor:ktor-auth:$ktor_version"
     implementation "io.ktor:ktor-websockets:$ktor_version"
-    compile "io.ktor:ktor-client-apache:$ktor_version"
+    implementation "io.ktor:ktor-client-apache:$ktor_version"
+    implementation "io.ktor:ktor-client-gson:$ktor_version"
     implementation "org.jetbrains.squash:squash:$squash_version"
     implementation "org.jetbrains.squash:squash-h2:$squash_version"
     implementation 'com.github.salomonbrys.kotson:kotson:2.5.0'
-    implementation "ch.qos.logback:logback-classic:1.2.1"
     implementation group: 'com.zaxxer', name: 'HikariCP', version: '2.7.2'
 
+    implementation 'ch.qos.logback:logback-classic:1.2.3'
+
+    testImplementation "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
+    testImplementation "org.jetbrains.kotlin:kotlin-test:$kotlin_version"
     testImplementation group: 'junit', name: 'junit', version: '4.12'
 }
 
-compileKotlin {
-    kotlinOptions.jvmTarget = "1.8"
-}
-compileTestKotlin {
-    kotlinOptions.jvmTarget = "1.8"
-}
+compileKotlin.kotlinOptions.jvmTarget = '1.8'
+compileTestKotlin.kotlinOptions.jvmTarget = '1.8'
 
 sourceSets {
-    main.java.srcDirs += "src"
-    main.resources.srcDirs += "resources"
-    main.kotlin.srcDirs += "src"
-    test.java.srcDirs += "test"
-    test.kotlin.srcDirs += "test"
+    main.java.srcDirs += 'src'
+    main.resources.srcDirs += 'resources'
+    main.kotlin.srcDirs += 'src'
+    test.java.srcDirs += 'test'
+    test.kotlin.srcDirs += 'test'
 }
 
 shadowJar {
     baseName = 'backend'
     classifier = null
     version = null
-}
\ No newline at end of file
+}
+
+kotlin.experimental.coroutines 'enable'
```

Comment:
You know this much better, but is it still valid as the coroutines are not experimental any more?